includes/admin/reporting/reports.php

```diff
@@ -2552,19 +2552,25 @@ function display_export_report() {
     <div id="edd-dashboard-widgets-wrap">
         <div class="metabox-holder">
             <div id="post-body">
-                <div id="post-body-content">
+                <div id="post-body-content" class="edd-reports-export">
 					<?php do_action( 'edd_reports_tab_export_content_top' ); ?>
 
                     <div class="postbox edd-export-earnings-report">
                         <h3 class="hndle"><span><?php esc_html_e( 'Export Earnings Report', 'easy-digital-downloads' ); ?></span></h3>
                         <div class="inside">
                             <p><?php esc_html_e( 'Download a CSV giving a detailed look into earnings over time.', 'easy-digital-downloads' ); ?></p>
                             <form id="edd-export-earnings" class="edd-export-form edd-import-export-form" method="post">
-								<?php echo EDD()->html->month_dropdown( 'start_month' ); ?>
-								<?php echo EDD()->html->year_dropdown( 'start_year' ); ?>
-								<span class="edd-label"><?php echo _x( 'to', 'Date one to date two', 'easy-digital-downloads' ); ?></span>
-								<?php echo EDD()->html->month_dropdown( 'end_month' ); ?>
-								<?php echo EDD()->html->year_dropdown( 'end_year' ); ?>
+								<div class="edd-to-and-from-container">
+									<?php echo EDD()->html->month_dropdown( 'start_month' ); ?>
+									<?php echo EDD()->html->year_dropdown( 'start_year' ); ?>
+								</div>
+
+								<span class="edd-label edd-export-to-label"><?php echo _x( '-- to --', 'Date one to date two', 'easy-digital-downloads' ); ?></span>
```

Comment:
I know it's just a class name, but it kind of bothers me to have `label` in the class name when this isn't a label--it's more of a spacer or descriptive (okay not descriptive because you can read it) element. What would you think about changing the class(es) to something else entirely and updating the CSS accordingly?

Reply:
We'll have to be careful here and make sure we don't cross-contaminate the front end since `edd-label` is used on the card fields (where they are `label` tags).

Reply:
Could I just make up a class and leave the `edd-label` as is?  Maybe `edd-export-to-spacing`?  Something more eloquent / creative?

Reply:
This `.edd-to-and-from-container` isn't anywhere else in the codebase, so maybe a variation on that? `.edd-to-and-from--separator` or something?

Reply:
I like that.  I can make that work with no `label`.